packages/uswds-elements/lib/_normalize.scss

```diff
@@ -1,16 +1,12 @@
-/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
-
 /* Document
    ========================================================================== */
 
 /**
- * 1. Correct the line height in all browsers.
- * 2. Prevent adjustments of font size after orientation changes in iOS.
+ * Correct the line height in all browsers.
  */
 
 html {
-  line-height: 1.15; /* 1 */
-  -webkit-text-size-adjust: 100%; /* 2 */
```

Comment:
It looks like `-webkit-text-size-adjust: 100%` is present in all three linked normalize files.

Was there another reason this was removed?

Reply:
Good question! It took me a minute to figure out what might have happened here, but I suspect I may have removed it since it wasn't present in all three at the original time I created the pull request. At least for `@csstools/normalize.css`, this was a more recent addition in December ([related issue](https://github.com/csstools/normalize.css/issues/25)). I can bring it back.

Reply:
Updated (restored) in 896efb66c02e31376b415dfe767ce9375ea9d07c.

Reply:
Awesome! Thanks for the update and the related links! 💯